app/javascript/components/patient/Patient.js

```diff
@@ -272,6 +272,25 @@ class Patient extends React.Component {
               <div>
                 <b>Primary Telephone Type:</b> <span>{this.props.details.primary_telephone_type || '--'}</span>
               </div>
+              {(this.props.details.secondary_telephone || this.props.details.international_telephone) && (
```

Comment:
I don't love the indentation and small text here.  not sure if there were mockups but thoughts on moving these to a second column (see how identification is done) and then stack them on small screens?

Reply:
If you input a secondary phone as well, you can see this is consistent with the existing styling

![image](https://user-images.githubusercontent.com/8235974/134553972-216962c0-1b07-41b4-a287-4a0b5c284d3d.png)

Reply:
What do you mean? None of the indented fields were displayed before.  We don't indent any other fields in the monitoree details and they're all the same font size.  I think it would be more readable and look better in two columns or just have them all the same style but just my opinion

Reply:
<img width="843" alt="Screen Shot 2021-09-23 at 1 32 35 PM" src="https://user-images.githubusercontent.com/35042815/134556110-a90d1b26-1ac8-4c5a-b876-41e37e93b615.png">

Reply:
Oh my bad I didn't realize the secondary phone indentation was new was well. I see now that before the secondary phone wasn't actually even displayed at all.

I actually prefer to leave it as Tim has implemented it, since it is consistent with the mockups and has been approved in UAT. Especially because the "Alternate Contact" designed in the mockups will go in the right hand column if/when that is implemented. Plus I like that the phone numbers are all grouped. If a user tries calling a monitoree and they don't answer. As designed, this makes it obvious there is a secondary and/or international number that the user can consider trying next.

Reply:
Ah the alternative contact reasoning makes sense.  I still think it looks weird based on the other sections look, so this could be an option as well 

<img width="326" alt="Screen Shot 2021-09-23 at 2 41 58 PM" src="https://user-images.githubusercontent.com/35042815/134565356-6afdaa15-10e6-4767-9b66-d6f452ce5b1b.png">